target_chains/near/receiver/src/ext.rs

```diff
@@ -11,3 +24,19 @@ pub trait Wormhole {
     /// Returns the Governance Index of the current GuardianSet only if the VAA verifies.
     fn verify_vaa(&self, vaa: String) -> u32;
 }
+
+/// An external definition of the Pyth interface.
+#[ext_contract(ext_pyth)]
+pub trait Pyth {
+    fn update_price_feed(&mut self, data: String) -> Result<(), Error>;
```

Comment:
Was named this way because originally it only updated one at a time unliek the Cosmos contract, to make refunding simpler, with accumulators we don't have this problem. I'll rename it.